scripts/create_istio_system.sh

```diff
@@ -31,5 +31,3 @@ else
   echo "running istioctl upgrade"
   istioctl upgrade -i $ISTIO_NAMESPACE -y ${ISTIO_FILES[@]/#/-f }
 fi
-istioctl manifest generate -i $ISTIO_NAMESPACE ${ISTIO_FILES[@]/#/-f } > /app/generated-manifest.yaml
-istioctl verify-install -i $ISTIO_NAMESPACE -f /app/generated-manifest.yaml
```

Comment:
should this be tracked in a separate PR or is it related to the 1.7.3 upgrade?

Reply:
Its not related, I can track in a different pr - I was thinking we would release 1.7.3-rancher1 and then overwrite it when adding this two line change in so it seemed easier to keep it in the say pr from a tagging perspective, but guess we could release it with tag 1.7.3-rancher2 and overwrite the image with just the version change

Reply:
either option sounds good to me, we can keep it the way it is since it's nbd imo

Reply:
We could also do separate PR's with 1 tag. But 1 pr 1 tag is fine for small changes like this.